pkg/ccl/streamingccl/replication_manager.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package streamingccl
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/kv"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/streaming"
+	"github.com/cockroachdb/cockroach/pkg/util/hlc"
+	"github.com/cockroachdb/errors"
+)
+
+type replicationStreamManagerImpl struct{}
+
+// CompleteStreamIngestionHook hooks a CompleteStreamIngestion implementation
+// inside streamingccl package.
+var CompleteStreamIngestionHook func(evalCtx *tree.EvalContext, txn *kv.Txn, jobID int, cutoverTimestamp hlc.Timestamp) error
+
+func (r replicationStreamManagerImpl) CompleteStreamIngestion(
```

Comment:
nit:
lets add method comments here and start method:

// CompleteStreamIngestion implements ReplicationStreamManager interface.